app/templates/index.html

```diff
@@ -299,6 +299,7 @@ <h5 class="white-text">LibreTranslate</h5>
 		  if (this.status >= 200 && this.status < 400) {
 			// Success!
 			self.langs = JSON.parse(this.response);
+			self.langs.push({ name: 'Auto (experimental)', code: 'auto' })
```

Comment:
This makes the auto option selectable only in the frontend. The api GET /languages does not contain the auto option.

Reply:
Yep, the issue is that LibreTranslate use the list loaded by Argos, and I don't want to change this Argos list, so we need to add `auto` manually in the UI and API lists (got to do the last one, thanks for the notice!)